packages/eslint-plugin/src/unsupported-route-components.ts

```diff
@@ -0,0 +1,49 @@
+import { ESLintUtils } from '@typescript-eslint/utils'
+
+const createRule = ESLintUtils.RuleCreator.withoutDocs
```

Comment:
One thing I should've asked before approving: do we need to add `@typescript-eslint/utils` to this package's deps?